ghost/core/core/server/data/migrations/versions/5.45/2023-04-18-12-56-add-announcement-settings.js

```diff
@@ -8,12 +8,6 @@ module.exports = combineTransactionalMigrations(
         flags: 'PUBLIC',
         group: 'announcement'
     }),
-    addSetting({
```

Comment:
noob question 🐢 : are we permitted to edit previous migrations before release?

Reply:
This goes by similar logic like "renaming the column" - [described here](https://www.notion.so/ghost/Database-migrations-eb5b78c435d741d2b34a582d57c24253?pvs=4#a1693683c4da4f42b39ca335e4d23f00). Removing this bit of migration is equivalent to "nooping" the migration.